ixmp4/__init__.py

```diff
@@ -1,16 +1,16 @@
-# flake8: noqa
 import importlib.metadata
 
+from ixmp4.core import IndexSet as IndexSet
```

Comment:
I'm wondering if we really need to import all of these objects at the top-level? I guess that except for Platform, we will never call them directly, right?

Reply:
I'm not sure if some downstream packages already use that syntax. We seem to only use it for our tests, where we regularly do `from ixmp4 import Variable`, `from ixmp4 import Model`, or `from ixmp4 import IndexSet`. I don't think this is necessary, but I also don't know what harm it's causing.

Reply:
There are no downstream packages yet, so better to keep the top-namespace lean.

Reply:
So out of all these imports, which would we want to keep? Only `Platform`?

Reply:
I'd at least not add more imports at this point, and we can do a separate remove-imports-PR later.

Reply:
But yes, I think that only `Platform` is the one that is already used - at least in **pyam**.

Reply:
Another useful one is "Datapoint" because we sometimes need the "Datapoint.Type" enum in user code. 
But i think its time we find a better solution for that anyway...